code/modules/admin/verbs/admin.dm

```diff
@@ -181,9 +181,9 @@ monkestation end */
 			qdel(W)
 			M.regenerate_icons()
 
-	log_admin("[key_name(usr)] made [key_name(M)] drop everything!")
-	var/msg = "[key_name_admin(usr)] made [ADMIN_LOOKUPFLW(M)] drop everything!"
-	message_admins(msg)
+	var/msg = "[key_name(usr)] made [key_name(M)] drop everything!"
+	log_admin(msg)
+	message_admins("[key_name_admin(usr)] made [ADMIN_LOOKUPFLW(M)] drop everything!")
```

Comment:
What's the purpose of reordering these lines...?

Reply:
Makes the code a bit better, log_admin and admin_ticket_log getting the same message, w/o html tags, and then message_admins gets the tags, so I needed to move the definition of msg above the log_admin